platform-includes/getting-started-config/javascript.react-router.mdx

```diff
@@ -0,0 +1,131 @@
+### Expose Hooks
+React Router exposes two hooks in your `app` folder (`entry.client.tsx` and `entry.server.tsx`).
+If you do not see these two files, expose them with the following command:
+
+```bash
+npx react-router reveal
+```
+
+### Client-Side Setup
+
+Initialize the Sentry React SDK in your `entry.client.tsx` file:
+
+```tsx {filename: entry.client.tsx} {"onboardingOptions": {"performance": "9, 12-16", "session-replay": "10, 17-21"}}
+import * as Sentry from "@sentry/react";
+import { startTransition, StrictMode } from "react";
+import { hydrateRoot } from "react-dom/client";
+import { HydratedRouter } from "react-router/dom";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [
+    Sentry.browserTracingIntegration(),
```

Comment:
shouldn't this use the react router integration?

Reply:
No as the routing mechanism is different when using framework mode. If you rely on the routes module then these functions that we need for our "library mode integration" are not exposed